ssg/build_remediations.py

```diff
@@ -276,9 +232,8 @@ def parse_from_file_with_jinja(self, env_yaml, cpe_platforms):
                 "    >&2 echo 'Remediation is not applicable, nothing was done'")
             wrapped_fix_text.append("fi")
 
-            remediation = namedtuple('remediation', ['contents', 'config'])
-            result = remediation(contents="\n".join(wrapped_fix_text), config=result.config)
-
+            result = namedtuple('remediation', ['contents', 'config'])(
+                contents="\n".join(wrapped_fix_text), config=result.config)
```

Comment:
I would define this named tuple separately at the beginning of the file.

Reply:
Done: https://github.com/ComplianceAsCode/content/pull/8923/commits/ad6aa4599bb9abc1a39a40a4e8b14dbfc215da2f